system/include/emscripten/posix_socket.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "websocket.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+extern EMSCRIPTEN_RESULT emscripten_init_websocket_to_posix_socket_bridge(const char *bridgeUrl);
+
+#ifdef __cplusplus
+}
+#endif
```

Comment:
why did `gl.symbols` change?

Reply:
I ran through the symbol update tool, which did changes to this file. Looks like old changes have not been committed. I'll drop the changes from this PR.